### ISSUE TITLE
Suport properly decoding mode A messages as mode C altitude when processing mode A data.

### ISSUE THREAD

**Issue by ThreeSixes** [ThreeSixes/airSuck#14]

**Comment by ThreeSixes** [comment]
The ssrParse class was updated with the addition of modeASquawk2modeCAlt() which decodes mode A squawk codes to a mode C altitude, assuming the input squawk code doesn't violate the constraints of a mode C altitude value. ssrParse()'s mode A/C decoder was updated to call this method when we have Mode A/C data. If the call to modeASquawk2ModeCAlt() doesn't return Null, we have a possible mode C altitude, and so we set 'cAlt' in the return data and leave 'aSquawk' in place.

**\* Note ***: We don't know if we're getting a mode A or C value since we don't get information about the interrogating RADAR's pulse spacing from the sensors. This just attempts to decode the response as a mode C altitude.
